mender/templates/api-gateway-configmap.yaml

```diff
@@ -340,6 +352,13 @@ data:
           loadBalancer:
             servers:
             - url: "http://{{ .Values.useradm.service.name }}:{{ .Values.useradm.service.port }}"
+{{- if .Values.api_gateway.minio.enabled }}
+
+        useradm:
```

Comment:
useradm becomes minio? (not quite familiar with that)

Reply:
typo